packages/functions_client/lib/src/types.dart

```diff
@@ -1,3 +1,4 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
```

Comment:
What's the reason for these ignores here?

Reply:
Sorry, not sure how it slipped in. Deleting it!